packages/rocketchat-ui-account/account/accountProfile.html

```diff
@@ -59,6 +59,9 @@
 				<div class="submit">
 					<button class="button"><i class="icon-send"></i><span>{{_ "Save_changes"}}</span></button>
 				</div>
+				<div class="logoutOthers">
+					<button class="button">Logout Others</button>	
```

Comment:
Can you translate this string?

Just replace `Logout Others` by `{{Logout_Others}}` and add this key and text in `i18n/en.i18n.json`